server/middlewares/db_validator.ts

```diff
@@ -20,7 +20,7 @@ export const userExist = async(req: Request, res: Response, next: NextFunction)
   next();
 };
 
-export const emailExist = async(req: Request, res: Response, next: NextFunction) => {
+export const emailNotExist = async (req: Request, res: Response, next: NextFunction) => {
   const { email } = req.body;
 
```

Comment:
An error happens if the user wants to leave the email in the current state (no modifying)